pkg/network/transport/object/grpc/replication.go

```diff
@@ -178,7 +180,20 @@ func (s *Server) Replicate(_ context.Context, req *objectGRPC.ReplicateRequest)
 		}}, nil
 	}
 
-	return new(objectGRPC.ReplicateResponse), nil
+	resp := new(objectGRPC.ReplicateResponse)
+	if req.GetSignObject() {
```

Comment:
do u plan to add tests for these cases? there are few of them now

Reply:
added some